x-pack/test/functional/page_objects/security_page.js

```diff
@@ -176,18 +176,13 @@ export function SecurityPageProvider({ getService, getPageObjects }) {
         const fullnameElement = await user.findByCssSelector('[data-test-subj="userRowFullName"]');
         const usernameElement = await user.findByCssSelector('[data-test-subj="userRowUserName"]');
         const rolesElement = await user.findByCssSelector('[data-test-subj="userRowRoles"]');
-        let reserved = false;
-        try {
-          reserved = !!(await user.setFindTimeout(500).findByCssSelector('[data-test-subj="reservedUser"]'));
-        } catch(e) {
-          //ignoring, just means user is not reserved
-        }
+        const isReservedElementVisible =  await user.findByCssSelector('td:last-child');
```

Comment:
😬 not a fan

Reply:
Not a huge fan either. EUI’s table doesn’t allow us to add attributes to the tds, we can only render what is inside of them, so in this situation we really don’t want to render anything when it’s not reserved. If we use the non-existence of the data-test-subject, this isn’t a good detector as something else could have caused that element to not be there.